src/post-submission-actions/program-enrollment-action.ts

```diff
@@ -33,26 +36,32 @@ export const ProgramEnrollmentSubmissionAction: PostSubmissionAction = {
             (enrollment) => enrollment.program.uuid === programUuid && enrollment.dateCompleted === null,
           );
           if (hasActiveEnrollment) {
-            throw new Error('Cannot enroll patient to program. Patient already has an active enrollment');
+            showToast({
+              title: t('errorSavingEnrollment', 'Error saving enrollment'),
+              kind: 'error',
+              critical: false,
+              description: t('cannotEnrollPatientToProgram', 'Cannot enroll patient to program. Patient already has an active enrollment'),
```

Comment:
I think these error messages could benefit from a design review.

Maybe something like the following reads better here:

```
Error enrolling to program

The patient is already enrolled to ${programName}
```